core/main/src/state/firebolt-open-rpc.json

```diff
@@ -6,52 +6,52 @@
 		"readable": "Firebolt Version Manifest v1.0.0"
 	},
 	"capabilities": {
-		"xrn:firebolt:capability:lifecycle:initialize": {
+		"xrn:firebolt:capability:accessibility:audiodescriptions": {
```

Comment:
Do we need to make any changes in `thor_permission_registry.yaml` file for updating the API/Capability mapping?